1-206-flowers/1-206-flowers.js

```diff
@@ -18,7 +21,11 @@ setInterval(() => {
   hostServer.emit('pickup', { event: 'pickup', box: box, time: Date.now });
 }, 5000);
 
+hostServer.on('intransit', (payload) => {
+  hostServer.emit('received', payload);
+});
 
 hostServer.on('delivered', (payload) => {
-  console.log('Thank you for delivering order# ', payload.box.orderID);
+  hostServer.emit('thankyou', payload);
+  hostServer.emit('received', payload);
```

Comment:
I would like to see all information that is being populated in driver and vendor added to the server